.github/workflows/gerrit-compose-required-rtdv3-verify.yaml

```diff
@@ -83,6 +83,8 @@ jobs:
         id: setup-python
         with:
           python-version: "3.8"
+      - name: Install graphviz
+        uses: tlylt/install-graphviz@main
```

Comment:
Please do not use `@main` but either a tag or better, the SHA related to the tag

Reply:
updated